daemon/wiretap_utils.go

```diff
@@ -18,6 +18,30 @@ func extractHeaders(resp *http.Response) map[string]any {
 	return headers
 }
 
+func reconstructURL(r *http.Request, protocol, host, port string) string {
+	url := fmt.Sprintf("%s://%s", protocol, host)
+	// pattern := "%s://%s:%s/%s?%s"
```

Comment:
I prefer to delete dead code. It's confusing, just remove rather than comment.

Reply:
Good catch. Forgot to remove it after the refactor. Updated now.